ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -110,12 +110,11 @@ class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate {
         case RPSampleBufferType.video:
             videoSource?.processVideoSampleBuffer(sampleBuffer)
             break
+
         case RPSampleBufferType.audioApp:
-            /*
-             * TODO: We do not capture app audio at the moment. For some broadcast use cases it may make sense to capture both the
-             * application and microphone audio. Doing this requires down-mixing the resulting streams.
-             */
+            ExampleCoreAudioDeviceRecordCallback(sampleBuffer)
             break
+
```

Comment:
Is there a way to ask replay kit to callback for app audio only vs microphone and vice versa?

Reply:
+1 to find out if this is possible in an extension. Another alternative is to check the static variable in each (audio) sample type case.

Reply:
All I could see was that you have the ability to turn on and off capture from the microphone in the ReplayKit App UI. If the mic button is on then I noticed that you sometimes get the low AVPlayer volume issue with other apps in the system. Fun stuff.